api/src/test/java/org/apache/iceberg/variants/TestSerializedObject.java

```diff
@@ -18,7 +18,7 @@
  */
 package org.apache.iceberg.variants;
 
-import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
```

Comment:
curious why do we change this. Seems `org.assertj.core.api.Assertions.assertThat;` is preferred one and it's not required to use AssertionsForClassTypes from your change.